test/test_service.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/python
+# coding=utf-8
+
+import json
+
+
+def test_download_mnist(mnist_class):
+    assert mnist_class.image_count == 10000
+
+
+def test_download_mnist_xshape(mnist_class):
+    assert mnist_class.x_test.shape == (10000, 28, 28)
+
+
+def test_download_mnist_yshape(mnist_class):
+    assert mnist_class.y_test.shape == (10000,)
+
+
+def test_model_loaded(mnist_class):
+    assert mnist_class.model is not None
+
+
+def test_session_initiated(mnist_class):
+    assert mnist_class.sess is not None
+
+
+def test_input_name(mnist_class):
```

Comment:
I expect this will also be tested by ONNX

Reply:
I wrote this test in as I have had trouble with not ensuring it is a string before, and the onnx error message was really unhelpful! So it's there as a guide for other devs